components/canvas/hooks/useVSMObjectMutation.ts

```diff
@@ -0,0 +1,59 @@
+import { useStoreDispatch } from "@/hooks/storeHooks";
+import { useProjectId } from "@/hooks/useProjectId";
+import { patchGraph } from "@/services/graphApi";
+import { notifyOthers } from "@/services/notifyOthers";
+import { NodeDataCommon } from "@/types/NodeData";
+import { debounce } from "@/utils/debounce";
+import { unknownErrorToString } from "@/utils/isError";
+import { useAccount, useMsal } from "@azure/msal-react";
+import { useMutation, useQueryClient } from "react-query";
+
+export const useVSMObjectMutation = (selectedNode: NodeDataCommon) => {
```

Comment:
We are moving away from the word VSM. I would suggest calling this useNodeUpdate, which is in line with useNodeAdd, useNodeMerge and useNodeDrag

Reply:
Fixed!